addons/core/functions/fnc_ClientInit.sqf

```diff
@@ -49,6 +49,8 @@ TF_respawnedAt = time;//first spawn so.. respawned now
         (findDisplay 46) displayAddEventHandler ["keyUp", "_this call TFAR_fnc_onLRTangentReleasedHack"];
 
         if (isMultiplayer) then {
+            (TFAR_currentUnit call TFAR_fnc_radiosList) apply { [_x, getPlayerUID player] call TFAR_fnc_setRadioOwner };            
```

Comment:
How do you make sure the player already got his Radios at this point?

Reply:
This code only need for JIP. JIP players don't receive OnRadiosReceived (not really sure) and radios not receiving correct owner. This code assigns owner to all radios in inventory when player joins after start.

Reply:
Owner should be automatically assigned when first using the Radio.
Also the RadioOwner is actually completly irrelevant. It's not used anywhere.

Reply:
If player joins in progress and don't use radio he will not receive comms.
Maybe I'm wrong.

Reply:
Yeah you are. Only thing that RadioOwner does. Is request a new Radio with a new ID if you happen to have a Radio in your inventory that someone else is already using.. If you get a Radio the owner is automatically set to your ID. If the owner is empty it is also automatically set to your ID. If you pick a Radio up from the ground the owner is automatically set to your ID. If you lay a radio on the ground the owner is automatically set to nothing.
So the Radio is only replaced if it the owner is somebody else. And that is impossible unless you give someone an already instanciated Radio with ID via a script although somebody else is already using that Radio. So unless you have scripts that screw something up there is no reason to have this.
And no it doesn't influence receiving comms at all.

Reply:
Great! So I can remove setRadioOwner everywhere except taking radio?

Reply:
Looks like owner is not used at all. Will remove all setRadioOwner calls.

Reply:
It is only used in the radio instanciation check. If you somehow managed to get a invalid radio into your inventory which can always happen by people using bad scripts it detects that and get's a replacement.